islands/RunButton.tsx

```diff
@@ -57,8 +67,9 @@ export default function RunButton({ id }: { id: string }) {
     <button
       class={tw`focus:outline-none`}
       onClick={run}
+      title={title}
```

Comment:
Maybe this can be a real tooltip so it can be viewed on mobile?

Reply:
Would you like to work on that? (or, can you please do that? I dont know how to lol)